README.md

```diff
@@ -1 +1,138 @@
-# First-Capstone-Project
+
+# First-Captone-Project
+
+# IT Audit
+
+This is a project acknowledging and affirming that I have properly understood everything learnt so far from the first module.
+
+> Learning objectives
+
+- I Used semantic HTML tags.
+- I Applied best practices in HTML code.
+- Used CSS selectors correctly.
+- Used CSS box model.
+- Used Flexbox to place elements on the page.
+- Demonstrated ability to create UIs adaptable to different screen sizes using media queries.
+- Used GitHub Pages to deploy web pages.
+- Applied JavaScript best practices and language style guides in code.
+- Used JavaScript to manipulate DOM elements.
+- Used JavaScript events.
+- Used objects to store and access data.
+- Communicated technical concepts to other technical people.
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+Click [here] [https://alabi12.github.io/First-Captone-Project/] to watch a walk-through description, given by me.
+
```

Comment:
- [ ] Kindly notice that as the previous code reviewer mentioned that you should include a loom video explaining and presenting your project, the video should not be more than 5 minutes long.